plugins/inputs/t128_metrics/t128_metrics_test.go

```diff
@@ -461,18 +463,79 @@ func TestT128MetricsRequestLimiting(t *testing.T) {
 	}
 
 	var acc testutil.Accumulator
-	plugin.Init()
+	require.NoError(t, plugin.Init())
 
 	require.NoError(t, acc.GatherError(plugin.Gather))
 }
 
+func TestTimoutUsedForRequests(t *testing.T) {
+	fakeServer := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		// hang request
+	}))
+
+	plugin := &plugin.T128Metrics{
+		BaseURL: fakeServer.URL,
+		ConfiguredMetrics: []plugin.ConfiguredMetric{{
+			"test-metric",
+			map[string]string{"test-field": "stats/test"},
+			map[string][]string{},
+		}},
+		Timeout:                 internal.Duration{Duration: 5 * time.Millisecond},
```

Comment:
The spacing seems off to me - is this a GO thing?

Reply:
Yeah, it's aligning to the rightmost `: <variable`, but it gets confused when one of the arguments is multiline like above. So, these last two values are being aligned.